src/types.ts

```diff
@@ -1,6 +1,6 @@
 import { DataQuery, DataSourceJsonData } from '@grafana/data';
 
-import { RequestSpec, defaultRequestSpec } from './RequestSpec';
+import { defaultRequestSpec, RequestSpec } from './RequestSpec';
```

Comment:
if the ci is ok with this change, we have to change the ci. import ordering should be stabilized.

Reply:
It's an CI error as well. See [here](https://github.com/tribe29/grafana-checkmk-datasource/actions/runs/3909744017/jobs/6681153601)